tcl/org.testeditor.tcl.dsl.ui.tests/src/org/testeditor/tcl/dsl/ui/tests/AbstractTest.xtend

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2015 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.tcl.dsl.ui.tests
+
+import com.google.inject.Guice
+import com.google.inject.Injector
+import com.google.inject.Module
+import java.util.ArrayList
+import java.util.List
+import javax.inject.Inject
+import org.eclipse.xtext.junit4.AbstractXtextTests
+import org.eclipse.xtext.junit4.XtextRunner
+import org.eclipse.xtext.junit4.validation.ValidationTestHelper
+import org.eclipse.xtext.util.Modules2
+import org.junit.runner.RunWith
+import org.testeditor.dsl.common.testing.AssertionHelper
+import org.testeditor.tcl.dsl.TclRuntimeModule
+import org.testeditor.tcl.dsl.TclStandaloneSetup
+
+@RunWith(XtextRunner)
+abstract class AbstractTest extends AbstractXtextTests {
```

Comment:
yeah, its already in org.testeditor.tcl.dsl.tests but neither test is a plugin project. the question is then how do I import the existing class...(e.g. make the tests plugin projects too)